project2/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Rubik:400,500&display=swap" rel="stylesheet">
+    <title>Movie Card</title>
+</head>
+
+<body>
+    <div>
+        <!--image was found on Google-->
+        <img src="images/YTuMamaTambien.jpg">
```

Comment:
Your image is missing `alt` text and you need to remember to resize and optimize images before your project is "final."